MCprep_addon/mcprep_ui.py

```diff
@@ -2007,6 +2106,10 @@ def register():
 		bpy.types.IMAGE_MT_uvs.append(mcprep_uv_tools)
 	# bpy.types.IMAGE_MT_image.append(mcprep_image_tools) # crashes, re-do ops
 
+	# Used to avoid checking for update file on disk too frequently.
+	global last_check_for_updated
```

Comment:
Wouldn't this be better off in `conf.py`?

Reply:
Good call, definitely better there